### DISCUSSION TITLE
Adjust the size of song lyrics

### DISCUSSION THREAD

**Discussion by AdsVector** [fast4x/RiMusic#2208] [Ideas]
I love the app and it would actually be great to be able to adjust the size of the letters. I wish I could choose bigger or smaller depending on the moment so I could read without getting too close.
I only ask that for subsequent updates.

**Comment by fast4x** [comment]
Hi, it's just implemented, look screenshot 

![Screenshot_2024-06-08-20-17-54-541_it.fast4x.rimusic-edit.jpg](https://github.com/fast4x/RiMusic/assets/1484476/f2f66630-965d-4ad6-b698-a651d132e68a)

**Reply by AdsVector** [reply]
Mmm, I already tried it with another device, if there is a small bug that does not show the box to select the size, it may be because the first device is a bit old.

Thanks for answering.

**Reply by fast4x** [reply]
You are welcome, try to enlarge album art in player by settings, would be show button.

https://github.com/fast4x/RiMusic/assets/1484476/698d6b82-dce2-49bf-b8d8-08014fbd646c
